gu-actix/src/release.rs

```diff
@@ -208,7 +208,7 @@ mod test {
         });
 
         use std::thread::sleep;
-        sleep(Duration::from_secs(5));
+        sleep(Duration::from_secs(1));
```

Comment:
why do we need 1 s here?
furthermore, few lines above you are using sleep from tokio_timer. Can't it be used here also?

Reply:
Q: "why do we need 1 s here?" 
A: it is a test ? sleep size is arbitrary. 
Q: " above you are using sleep from tokio_timer. Can't it be used here also?"
A: pls. read this test again. what we testing ? whats is diffrerence betweem this places.